scraping_wikidata.py

```diff
@@ -0,0 +1,59 @@
+import requests
+
+header = {
+	
+	'yukun tan': 'wikidata scraping'
+}
+import pandas as pd
+import sqlite3
+
+# query from wikidata, followed the tutorial
+
+query = '''
```

Comment:
The formatting of some of this is not totally PEP8 compatible. Also, maybe in the future you'll want to create some functions instead of a single-execution script?